### PR TITLE
feat: add information on tokens and sections

### LINKED ISSUE #28636: Issue with explanations, and  order of tokens
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

While I was working on a PR to solve this [issue](https://github.com/mdn/content/issues/20180), @dipikabh pointed out that there were some organizational issues. 

### What did you expect to see?

Fuller explanations 

### Do you have any supporting links, references, or citations?

https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by CBID2**
### Description
This PR adds a new section to the `autocomplete` attribute documentation, explaining the usage of tokens and sections. It provides examples of how to use this component to specify autofill details for different sections of a form. 

### Motivation
This addition would help readers gain a better understanding of how to effectively use the `autocomplete` attribute. 

### Additional details

This is the page that I'm working with: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete 
### Related issues and pull requests
#28636 
Relates to #20180

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @dipikabh, I made the PR! :)

**Review by estelle — COMMENTED** [review]
Thank you for the contribution.

I put substantial effort into this review, providing both suggested edits and explanations for some of those suggestions. Please read them all. We truly appreciate your contributions to MDN; please accept some of my comments as guidance for future PRs as well.

File: files/en-us/web/html/attributes/autocomplete/index.md [hunk 1]
```diff
@@ -159,6 +159,74 @@ See the [WHATWG Standard](https://html.spec.whatwg.org/multipage/forms.html#auto
...
+## Tokens and Sections
+
+If the autocomplete attribute is specified, there must be a value that contains an ordered set of [space-separated tokens](https://html.spec.whatwg.org/#set-of-space-separated-tokens) that consists of only autofill detail tokens. These tokens should be placed in the following order:
+
+1. Start with `section-`
+2. Use values like "shipping" or "billing" for addresses
+3. Use values like "home", "mobile", "work", "fax", or "pager" for contact numbers
+4. A required token with an autocomplete field. See [W3's standard](https://www.w3.org/WAI/standards-guidelines/act/rules/73f2c2/2022-01-28/#correct-autocomplete-field) on this topic to learn more.
+
+> **NOTE:** Numbers 1-3 are optional tokens.
```
[the remark this answers is not in the document]
Comment (Josh-Cena) [reply files/en-us/web/html/attributes/autocomplete/index.md:171]: Also, this needs to be `> **Note:**` to be a note.

File: files/en-us/web/html/attributes/autocomplete/index.md [hunk 2]
```diff
@@ -159,6 +159,74 @@ See the [WHATWG Standard](https://html.spec.whatwg.org/multipage/forms.html#auto
 </div>
 ```
 
+## Tokens and Sections
+
+If the autocomplete attribute is specified, there must be a value that contains an ordered set of [space-separated tokens](https://html.spec.whatwg.org/#set-of-space-separated-tokens) that consists of only autofill detail tokens. These tokens should be placed in the following order:
+
+1. Start with `section-`
```
[the remark this answers is not in the document]
Comment (CBID2) [reply files/en-us/web/html/attributes/autocomplete/index.md:166]: Hi @estelle! :) Based on what I've researched, the `section-` is supposed to be a string of some sort. I had trouble explaining this part in a digestible way, so I felt this was the best approach

File: files/en-us/web/html/attributes/autocomplete/index.md [hunk 3]
```diff
@@ -159,6 +159,74 @@ See the [WHATWG Standard](https://html.spec.whatwg.org/multipage/forms.html#auto
...
+
+```html
+<fieldset>
+  <legend>Send the red velvet cake to...</legend>
+  <p>
+    <label>
+      Address 1:
+      <textarea
+        name="rvc"
+        autocomplete="section-red velvet shipping street-address"></textarea>
```
[the remark this answers is not in the document]
Comment (CBID2) [reply files/en-us/web/html/attributes/autocomplete/index.md:182]: Hi @estelle! I was heavily inspired by the code snippet in the section [4.10.18.7.1](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute) on the HTML specification website.  I did `velvet` and `lemon` to distinguish the two shipping addresses. I hope that clarifies things

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/html/attributes/autocomplete/index.md:182]: The question is, what does the ident `velvet` is the space separated list of tokens within the attribute do.

[the remark this answers is not in the document]
Comment (CBID2) [reply files/en-us/web/html/attributes/autocomplete/index.md:182]: Hi @estelle.  'velvet` in the `autocomplete="section-red velvet shipping street-address` attribute is there to help the user to fill out the address information for the red velvet cake order in a quicker way.

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/html/attributes/autocomplete/index.md:182]: If you include something in the documentation, it needs to be explained to the reader. The "velvet" in this case has no meaning. I was not asking for my own edification. That said, how does it help the user to fill out the address information for the red velvet cake order in a quicker way? Does it? How does it fit in with all the other space-separated values within the autocomplete attribute value?

[the remark this answers is not in the document]
Comment (CBID2) [reply files/en-us/web/html/attributes/autocomplete/index.md:182]: > If you include something in the documentation, it needs to be explained to the reader. The "velvet" in this case has no meaning. I was not asking for my own edification. That said, how does it help the user to fill out the address information for the red velvet cake order in a quicker way? Does it? How does it fit in with all the other space-separated values within the autocomplete attribute value?

Ahh I see @estelle. Thanks for the questions. It's been quite difficult finding information and explaining how this works.

**Comment by CBID2** [comment]
> Hi @CBID2 - it looks like this PR has stalled since last September, which is usually an indicator that it should be closed. There's also a linked PR above which might have some overlap with yours. Should we close it for now? Do you need to discuss the changes on Discord? Thanks :)

Hey @bsmth. I'm open to discuss it on Discord

**Comment by bsmth** [comment]
Going to close this one for now to revisit in future

**Pull request closed without merging**